src/abstractdataframe/sort.jl

```diff
@@ -336,13 +336,40 @@ Sort.defalg(df::AbstractDataFrame, o::Ordering; alg=nothing, cols=[]) =
              lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
 
 Test whether data frame `df` sorted by column(s) `cols`.
+If column selector selects no columns, it checks if the `df` is sorted lexicographically.
```

Comment:
```suggestion
If `cols` selects no columns, check whether `df` is sorted on all columns.
```
I'd also put this below the next sentence to keep the first description as simple as possible.

Reply:
Ah, if the idea was to mention that sorting is done lexicographically, this could be mentioned here, but without referring to the case were `cols` selects no columns, as it applies more generally. It would also make sense to explain what "lexicographically" means exactly as not everybody is familiar with this term.